tasks/runtime.yml

```diff
@@ -71,12 +71,12 @@
   service:
     name: munge
     enabled: "{{ openhpc_slurm_service_enabled | bool }}"
-    state: "{{ 'started' if openhpc_slurm_service_enabled | bool else 'stopped' }}"
+    state: "{{ 'started' if openhpc_slurm_service_started | bool else 'stopped' }}"
 
 # In case the service isn't running and the config hasn't changed to trigger
```

Comment:
Doesn't this also need `enabled: "{{ openhpc_slurm_service_enabled | bool }}"`?

Reply:
Done. I've removed the separate task to enable the service which also restarted the service unnecessarily.